Clarify API application ownership in Entra ID authentication guide (#8460)
Particular/docs.particular.net: servicecontrol/security/entra-id-authentication.md

| Admin consent display name | `Full access to ServiceControl API`          |
| Admin consent description  | `Allows ServicePulse to call ServiceControl` |

> [!NOTE]

> > Ensure that an owner is [assigned to the API application](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-configure-app-expose-web-apis#assign-application-owner), otherwise the API will not be listed when requesting API permissions.

Reviewer: for these instructions to work, the current user would need to be made the owner, not some other random user. 
```suggestion
> Ensure that you are assigned as the [owner of the API application](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-configure-app-expose-web-apis#assign-application-owner), otherwise the API will not be listed when requesting API permissions.
```

alternatively, the instructions could change to say go to "all applications", rather than "my applications"
